assets/tabs.css

```diff
@@ -285,4 +285,134 @@ p {
     border-radius: 0.25rem;
     border: 0.5px solid lightgrey;
     z-index: 1000;
+}
+
+/* side bar */
+#sidebar {
```

Comment:
please change this. We should not be using all this CSS, just use it if stricly necessary. This is not necessary and will need to be removed